include/bm/bm_sim/extern.h

```diff
@@ -65,6 +65,16 @@ class ExternFactoryMap {
            #extern_name,                                                \
            [](){ return std::unique_ptr<ExternType>(new extern__()); });
 
+#define BM_REGISTER_EXTERN_TYPE(extern_name)                               \
```

Comment:
Seems redundant since BM_REGISTER_EXTERN already exists, but I created this to suppress a compiler warning where BM_REGISTER_EXTERN has an unused variable.